index.js

```diff
@@ -0,0 +1,60 @@
+import express from 'express'
+
+import {validateNewNote} from './validation/postNote.js';
+import {validateUpdateNote} from './validation/putNote.js';
+
+const app = express();
+const PORT = process.env.PORT ?? 3000;
+const ERROR_STATUS = 400;
+
+app.use(express.json());
+
+app.get('/api/greetings', (request, response) => {
+  const { name } = request.query;
+  if (name) {
+    response.send(`<p>Hello ${name}!</p>`)
+  } else {
+    response.status(ERROR_STATUS).json({ error: 'Name undefined' })
+  }
```

Comment:
Amazing, but let's update structure of our project. Seems pretty uncomfortable to define all the controllers here, inside one file:
1. We should create 2 types of routers: `notes` and `greetings`. So, we can create a separate file: `src/modules/greetings/routes`. And inside index.js provide greetings routes. The same for notes.
2. Create folder `src/modules/greetings/controllers`. Inside please define index.js which exporting all the controllers from this folder. And also `getName.js`. Inside getName.js you should provide all the functions/action for provided route:
```
const { name } = request.query;
  if (name) {
    response.send(`<p>Hello ${name}!</p>`)
  } else {
    response.status(ERROR_STATUS).json({ error: 'Name undefined' })
  }
```
3. The same for notes. Create an appropriate folders(Example: src/modules/notes/controllers/getNotes.js or src/modules/notes/controllers/createNote.js)